src/test_bls12_381.rs

```diff
@@ -9,14 +9,14 @@ use rustacuda_core::DevicePointer;
 use std::ffi::{c_int, c_uint};
 
 extern "C" {
-    fn poseidon_multi_cuda_bls12_381(
-        input: *const ScalarField_BLS12_381,
-        out: *mut ScalarField_BLS12_381,
-        number_of_blocks: usize,
-        arity: c_uint,
-        device_id: usize,
-        stream: usize, // TODO: provide a real stream
-    ) -> c_uint;
+    // fn poseidon_multi_cuda_bls12_381(
```

Comment:
why is all this commented out?

Reply:
>I had to comment out poseidon_multi_cuda_bls12_381 function and corresponding test because there's no extern equivalent on the CUDA side

Reply:
@DmytroTym It seems there is a corresponding method in cuda that is extern "C": https://github.com/celer-network/icicle/blob/main/icicle/curves/bls12_381/poseidon.cu#L10

Can we uncomment these functions and the test?

Reply:
Ah, you're right, I think the issue is just that `poseidon.cu` is not included into `supported_operations.cu` for bls12-381. Does it make sense @ChickenLover ? Am trying to include and rebuild now. Since we're fixing Rust tests, can I also `#[ignore]` the currently failing bls12-377 tests, until we fix it? It's an eyesore :(